sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.1.1 (2021-06-07)
+- Dependency versions updated.
```

Comment:
Isn't there an actual update to the linkedin fix? This comment doesn't seem right because we are not pushing this release for a core update

Reply:
I think this one the actually right. common moved from 1.0.1 to 1.0.2. Can you please elaborate on this, "we are not pushing this release for a core update"?

Reply:
The term "dependency versions updated" was use last time because the underlying Azure Core/Identity version were updated because of which we pushed out a release. But this time I thought you actually made a fix for the linkedin issue so shouldn't that be described here? Or was that fix in common instead, in which case you should update the changelog of common correctly.

Reply:
Either way, I think we should be saying Updated `azure-communication-identity` version because thats the generic pattern we have been following ..